Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1276,7 +1270,7 @@
     <RegEx find="\b([sS])(ljed)(a|e|i|u|om|im|iš|imo|ite|e|ili)?\b" replaceWith="$1lijed$3" />
     <RegEx find="([rR])(edosljed)(a|e|u|om)?" replaceWith="$1edoslijed$3" />
     <RegEx find="([sS])(meh)(a|u|om)?" replaceWith="$1mijeh$3" />
-    <RegEx find="([sS])(mešn)(a|e|i|o|u|om|oj|ima)" replaceWith="$1miješn$3" />
+    <RegEx find="([sS])(mj?ešn)(a|e|i|o|u|om|oj|ima)" replaceWith="$1miješn$3" />
```

Comment:
You don't need to capture second group, and replace `$1miješn$3` with `$1miješn$2`